Streetcode/Streetcode.BLL/MediatR/Streetcode/RelatedFigure/Delete/DeleteRelatedFigureHandler.cs

```diff
@@ -0,0 +1,30 @@
+using FluentResults;
+using MediatR;
+using Streetcode.DAL.Repositories.Interfaces.Base;
+
+namespace Streetcode.BLL.MediatR.Streetcode.RelatedFigure.Delete;
+
+public class DeleteRelatedFigureHandler : IRequestHandler<DeleteRelatedFigureCommand, Result<Unit>>
+{
+    private readonly IRepositoryWrapper _repositoryWrapper;
+
+    public DeleteRelatedFigureHandler(IRepositoryWrapper repositoryWrapper)
+    {
+        _repositoryWrapper = repositoryWrapper;
+    }
+
+    public async Task<Result<Unit>> Handle(DeleteRelatedFigureCommand request, CancellationToken cancellationToken)
+    {
+        var relation = await _repositoryWrapper.RelatedFigureRepository.GetFirstOrDefaultAsync(rel => rel.ObserverId == request.ObserverId && rel.TargetId == request.TargetId);
```

Comment:
Could you move such a long LINQ line to the next line, like: 
....RelatedFigures
.GetFirst()...
?

Reply:
good idea! thanks!